_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+
+Large city blocks are a common theme in the West. These typically vary between 198 feet, 330 feet, and 660 feet (which is 3, 5, and 10 chains). Often, two sides of the block are 660 feet, with the adjacent sides being 198 feet or 330 feet. Looking at Portland (Fig 9), their blocks are unusually small in the West at 198 feet x 198 feet. On the other side of the spectrum, Salt Lake City (Fig 9) is known for having the largest blocks in the West at 660 feet x 660 feet. Blocks are more rectangular in San Francisco (Fig 9) and Phoenix (Fig 9), as they measure 660 feet x 198 feet and 660 feet x 330 feet, respectfully.
+
+<div class="caption">
+  <a href="{{ "/images/urban_grids.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/urban_grids_sm.png" | prepend: site.baseurl }}" alt="Western Urban Grid">
+  </a>
+  <p class="caption-text">Fig 9: Variations on the Western Urban Grid</p>
+</div>
+
+If you have ever tried to cross the street in downtown Salt Lake City, you’ll know our intersections are extremely wide. You’ll see the Gunter Chain’s lasting mark at these intersections as well. Looking at Portland (Fig 9) and Salt Lake City (Fig 9) again and you’ll notice two variations of the chain. Salt Lake City (Fig 9) has large intersections that are based on two chain lengths (132 feet), whereas in Portland and San Francisco intersections measure half a chain (33’). In Phoenix (Fig 9), you’ll notice intersections measure one full chain length (66 feet).
+
+## Section Lines and Addressing
+{: .text-left}
+
+Another notable mention are “section line roads”. These roads are laid out in a one-mile by one-mile grid pattern that follow the section lines. Figure 10 shows a good example of this southwest of Boise, Idaho. This pattern is also very popular in Phoenix and Las Vegas.
+
+Section lines in North Dakota and South Dakota are often used as the basis for their street numbering systems. In Salt Lake City, a similar section-based addressing system is used, where the 100 blocks (ex: from 100 E to 200 E) are ten chains, or 660 feet, or quarter-quarter-quarter sections (10 chains or 660 feet).
+
+One last notable Western development pattern is the disguised grid. These are often large communities. The land is typically purchased by the section (or more) and then made to feel less grid-like. Two good examples of this are Sun Lakes (Fig 11) and Sun City in Arizona. As you move through these communities you forget that you're on the grid.
+
+<div class="caption">
+  <a href="{{ "/images/SectionLineRoads.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/SectionLineRoads_sm.png" | prepend: site.baseurl }}" alt="Section Line Roads Southwest of Boise, ID">
+  </a>
+  <p class="caption-text">Fig 10: Section Line Roads Southwest of Boise, ID</p>
```

Comment:
And again with the fig 10 on left, fig 11 on right

Reply:
@steveoh do you know how to do that?  I spent a bunch of time trying to figure that, too and I got nowhere.  I settled on just leaving it.

Reply:
yes I know how to do that.